backend/router/page_crud.py

```diff
@@ -52,7 +54,13 @@ def update_page(page_id: int, page: PageUpdate, db: Session = Depends(get_db)):
     if db_page is None:
         raise HTTPException(status_code=404, detail="Page not found")
 
-    db_page.html_content = page.html_content
+    if page.title is not None:
+        db_page.title = page.title
+    if page.html_content is not None:
+        db_page.html_content = page.html_content
+    if page.category_id is not None:
+        db_page.category_id = page.category_id
```

Comment:
Existuje aj prípad kedy môžeš nešpecifikovať jednu z týchto hodnôt a stále sa to uloží do databázy?

Podľa mňa by mali byť všetky atribúty required (povinné) na úrovni pydantic modelu, potom sa automaticky aj validujú a nemusíš tu mať tieto riadky kódu